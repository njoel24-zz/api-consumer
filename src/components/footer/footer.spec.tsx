import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './footer.component';

describe('Footer', () => {
	it('renders component and compare with snapshot', () => {
		const component = renderer.create(<Footer/>);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});