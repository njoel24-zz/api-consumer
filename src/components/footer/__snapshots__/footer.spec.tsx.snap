// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders component and compare with snapshot 1`] = `
<div>
  <h4>
    Data Provided by IQAir
  </h4>
</div>
`;
